common/components/BalanceSidebar/AddCustomTokenForm.jsx

```diff
@@ -0,0 +1,99 @@
+// @flow
+import React from 'react';
+import { isValidETHAddress, isPositiveIntegerOrZero } from 'libs/validators';
+import translate from 'translations';
+
+export default class AddCustomTokenForm extends React.Component {
+  props: {
+    onSave: ({ address: string, symbol: string, decimal: number }) => void
```

Comment:
decimals is correct spelling, it is decimal here because it is that way in our token json